server/src/test/java/se/fortnox/reactivewizard/server/RwServerTest.java

```diff
@@ -204,6 +205,21 @@ void shouldAwaitShutdownDependency() {
         ));
     }
 
+    @Test
+    void shouldAwaitShutdownLogging() {
```

Comment:
Could we also add one test verifying the handling of a shutdown timeout?

Reply:
Good idea :)

Reply:
This and @jepp3:s comment above is resolved by not having a timeout anymore, right?

Reply:
This is not relevant any longer. The timeout has been removed.